src/main/java/tk/shanebee/hg/listeners/GameListener.java

```diff
@@ -418,8 +420,15 @@ private void onInteract(PlayerInteractEvent event) {
         } else if (action != Action.PHYSICAL && playerManager.hasPlayerData(player)) {
             Status status = playerManager.getPlayerData(player).getGame().getGameArenaData().getStatus();
             if (status != Status.RUNNING && status != Status.BEGINNING) {
-                event.setCancelled(true);
-                Util.scm(player, lang.listener_no_interact);
+				if(event.getItem().getType() == Material.RED_BED){
```

Comment:
There should be spacing between if and (, as well as between ) and {
```suggestion
				if (event.getItem().getType() == Material.RED_BED) {
```
Also, as said before the item should be configurable